### ISSUE TITLE
Three dependencies in installation_doconce are not available anymore

### ISSUE THREAD

**Issue by aless80** [doconce/doconce#46]
The following lines in the installation script `doc/src/manual/install_doconce.sh` fail (**cloud_sptheme, sphinxjp.themes.solarized, pygments-ipython-console**):

```
pip_install --exists-action i -e hg+https://bitbucket.org/ecollins/cloud_sptheme#egg=cloud_sptheme

Obtaining cloud_sptheme from hg+https://bitbucket.org/ecollins/cloud_sptheme#egg=cloud_sptheme
  Cloning hg https://bitbucket.org/ecollins/cloud_sptheme to /home/amarin/doconce/venv/src/cloud-sptheme
abort: HTTP Error 404: Not Found
Command "hg clone --noupdate -q https://bitbucket.org/ecollins/cloud_sptheme /home/amarin/doconce/venv/src/cloud-sptheme" failed with error code 255 in None
```
```
pip install --upgrade --exists-action i -e hg+https://bitbucket.org/miiton/sphinxjp.themes.solarized#egg=sphinxjp.themes.solarized
Obtaining sphinxjp.themes.solarized from hg+https://bitbucket.org/miiton/sphinxjp.themes.solarized#egg=sphinxjp.themes.solarized
  Cloning hg https://bitbucket.org/miiton/sphinxjp.themes.solarized to /home/amarin/doconce/venv/src/sphinxjp.themes.solarized
abort: HTTP Error 404: Not Found
Command "hg clone --noupdate -q https://bitbucket.org/miiton/sphinxjp.themes.solarized /home/amarin/doconce/venv/src/sphinxjp.themes.solarized" failed with error code 255 in None
```

```
pip install --upgrade --exists-action i -e git+https://bitbucket.org/doconce/pygments-ipython-console#egg=pygments-ipython-console
Obtaining pygments-ipython-console from git+https://bitbucket.org/doconce/pygments-ipython-console#egg=pygments-ipython-console
  Cloning https://bitbucket.org/doconce/pygments-ipython-console to /home/amarin/doconce/venv/src/pygments-ipython-console
remote: Repository doconce/pygments-ipython-console not found
fatal: repository 'https://bitbucket.org/doconce/pygments-ipython-console/' not found
Command "git clone -q https://bitbucket.org/doconce/pygments-ipython-console /home/amarin/doconce/venv/src/pygments-ipython-console" failed with error code 128 in None

```

Similar errors takes place in the other installation file `doc/src/manual/install_doconce.py`. 

The dependencies have probably moved, for example to [cloud-sptheme.readthedocs.io](https://cloud-sptheme.readthedocs.io/en/latest/install.html), but they should be tested

**Comment by MartinHeroux** [comment]
Hi Alessandro,

I can have a look at this and see if I can find out where they have moved to.

**Comment by aless80** [comment]
cool, thank you

**Comment by aless80** [comment]
Fixed by #48
